pipelines/collate_plots.py

```diff
@@ -136,17 +153,24 @@ def process_dir(
         named subdirectories. If ``None``, look in all
         subdirectories of ``base_dir``. Default ``None``.
     """
+    if save_dir is None:
+        save_dir = Path(base_dir).parent / "figures"
```

Comment:
Create if not exists?

Reply:
This function calls `merge_pdfs_from_subdirs`, which makes the directory if it doesn't exist.